### ISSUE TITLE
pirsf017184

### ISSUE THREAD

**Issue by carighi** [pircurators/tracking#18]
I am looking at members of PIRSF pirsf017184
members are bifunctional proteins

However, the entries are annotated with this PIRNR and also are annotated by HAMAP but the HAMAP annotation is via 2 rules
HAMAP-Rule:MF_01966 and HAMAP-Rule:MF_01965

See this example:
http://www.uniprot.org/uniprot/E1V682

Because proteins are bifunctional is hitting 2 of the HAMAPs and adding annotation from both rules.
This causes annotation issues in UniRule. For example, because the names proposed by PIRNR rule which is correct(has the include with the two domains and names) differs from the HAMAP ones, then the names are not used.

I sent email to Ivo and Elisabeth at SIB to fix this.

**Comment by carighi** [comment]
email from SIB04/20/2018
Hi Cecilia,

thanks for pointing this case out, it's something we haven't considered
yet I think.

I fear there is nothing easy to fix on our side. It is actually not the
HAMAP rule(s) producing this annotation, but an implementation Vladimir
has done for non-overlapping, multiple signature-matching proteins in
TrEMBL to increase the coverage of annotations. HAMAP by itself does not
annotate fusion proteins currently.

So in short, Vladimir produces an (artificial = not from the rules)
RecName for the fusion protein and uses the different rule RecNames to
fill "Includes" sections in the matching multifunctional protein. We
haven't thought about the case where there would be a rule for the
fusion protein itself and how to handle this.

Indeed something to think about

Best

-ivo
